File: mobile/src/main/java/technology/mainthread/apps/gatekeeper/view/fragment/ComingSoonFragment.java
package technology.mainthread.apps.gatekeeper.view.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.trello.rxlifecycle2.components.support.RxFragment;

import technology.mainthread.apps.gatekeeper.R;

public class ComingSoonFragment extends RxFragment {

    public static Fragment newInstance() {
        return new ComingSoonFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_coming_soon, container, false);
    }
}

File: mobile/src/main/java/technology/mainthread/apps/gatekeeper/view/fragment/ComingSoonFragment.kt
package technology.mainthread.apps.gatekeeper.view.fragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import technology.mainthread.apps.gatekeeper.R

fun buildComingSoonFragment(): Fragment {
    return ComingSoonFragment()
}

class ComingSoonFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_coming_soon, container, false)
    }

}
